main.py

```diff
@@ -12,7 +12,7 @@
 from pathlib import Path
 from twitchbot import BaseBot
 from twitchbot import event_handler, Event, Command, Message, Channel, PollData, get_bot
-from twitchbot.config import get_nick, get_oauth, get_client_id, get_client_secret
+from twitchbot.config import get_nick, get_oauth, get_client_id
```

Comment:
`get_client_secret` seems to be unavailable in the version of `PythonTwitchBotFramework` that I use, and this import seems to be unused and overriden a few lines down anyway.